### ISSUE TITLE
Add a detailed Contributing doc

### ISSUE THREAD

**Issue by alensiljak** [orgzly-revived/orgzly-android-revived#45]
A more fleshed-out Contributing doc is needed for the project.
For the moment I've created a section in the Readme, just to cover the very basics.

**Comment by amberin** [comment]
Great. I've had the same thought - this is important.

**Comment by alensiljak** [comment]
Yes, now that we are publishing on F-Droid (almost!), the public-facing things become important. This is the first thing people see and establishes the feel for the project. I'm not saying it is urgent, as it is still a hobby project, but it is important.
